lib/driver.js

```diff
@@ -668,11 +655,11 @@ export class XCUITestDriver extends BaseDriver {
       await this.activateRecentWebview();
     }
     if (this.isSafari()) {
-      if (shouldSetInitialSafariUrl(this.opts) && !this._didOpenInitialSafariUrlViaWda) {
-        this.log.info(
-          `About to set the initial Safari URL to '${this.getCurrentUrl()}'.` +
-            `Use 'safariInitialUrl' capability in order to customize it`,
-        );
+      if (shouldSetInitialSafariUrl(this.opts)) {
+        this.log.info(`About to set the initial Safari URL to '${this.getCurrentUrl()}'`);
+        if (_.isNil(this.opts.safariInitialUrl) && !_.isNil(this.opts.initialDeeplinkUrl)) {
```

Comment:
Should this be `!_.isNil(this.opts.initialDeeplinkUrl)`? The extra sentence is then only shown if `appium:initialDeeplinkUrl` is set, but the docs mention that setting `appium:initialDeeplinkUrl` causes `appium:safariInitialUrl` to be ignored anyway.

Reply:
good catch. The log message should actually be shown if both caps are unset